node/src/p2p/shwap.rs

```diff
@@ -34,23 +34,13 @@ impl Multihasher<MAX_MH_SIZE> for ShwapMultihasher {
 pub(super) fn row_cid(row_index: u16, block_height: u64) -> Result<Cid> {
     let row_id = RowId::new(row_index, block_height).map_err(P2pError::Cid)?;
 
```

Comment:
could you remove also the remaining empty lines? I don't think they add readability for so short functions